### ISSUE TITLE
Create App Icon

### ISSUE THREAD

**Issue by blchelle** [CMPUT301F20T41/boromi#25]
## Description
This isn't really a necessary feature of the application but it is a "nice to have" feature that won't take a lot of time to complete. Go ahead and work on this if there are no other issues to work on or if all your issues are blocked by something else

## Deliverables
1. I've already gone ahead and designed a simple icon based on the logo, simply download the png file from the following link and use it in step 2. [App Icon](https://user-images.githubusercontent.com/39068407/97489046-aa343f80-1924-11eb-8625-d9e8271752ae.png)
2. Follow this page https://developer.android.com/studio/write/image-asset-studio
